packages/patrol/android/src/main/kotlin/pl/leancode/patrol/PatrolJUnitRunner.java

```diff
@@ -1,3 +1,7 @@
+// Terminology note:
+// "Run a test" is used interchangeably with "execute a test".
+// "Run a Dart test" is used interchangeably with "request execution of a Dart test" and "execute Dart test".
+
```

Comment:
This also applies to the iOS part. I wonder what could be the best place to put such things. Maybe some `IMPLEMENTATION.md` in the (currently non-existent) `docs` directory in repo root?